src/app/EventLoggingTypes.h

```diff
@@ -140,6 +141,7 @@ class EventOptions
     PriorityLevel mPriority = PriorityLevel::Invalid;
     Type mUrgent            = Type::kNotUrgent; /**< A flag denoting if the event is time sensitive.  When kUrgent is set, it causes
                                                        the event log to be flushed. */
+    Optional<FabricIndex> mFabricIndex;
```

Comment:
Should document the semantics of this member, especially what it means to have or not have a value.

Reply:
just remove optional, thanks

Reply:
documented